packages/expo-haptics/src/Haptics.ts

```diff
@@ -3,6 +3,15 @@ import { UnavailabilityError } from '@unimodules/core';
 import ExpoHaptics from './ExpoHaptics';
 import { NotificationFeedbackType, ImpactFeedbackStyle } from './Haptics.types';
 
+/**
+ * Returns whether the Haptics API is enabled on the current device. This does not check the app permissions.
+ */
+export async function isAvailableAsync(): Promise<boolean> {
+  return (
+    !!ExpoHaptics.notificationAsync && !!ExpoHaptics.impactAsync && !!ExpoHaptics.selectionAsync
```

Comment:
Well, I'm not sure about it here 🤔 
Let's say that a mobile device exposes these methods natively to JS side (because the `expo-haptics` is included in the project), but for hardware reason these methods do nothing. Then this would cause misinformation, wouldn't it?
I would rather stick to the scenario that we expose native method and check the API availability natively.

Reply:
Yeah, I agree with this one! Vibration is something we can check pretty easily on the native side as well. I'll try to update the PR today with a native implementation 😄